tests/testthat/test-shinytest2-landing_popup.R

```diff
@@ -0,0 +1,184 @@
+testthat::test_that("e2e: teal app with landing_popup_module initializes with no errors", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = modules(
+      landing_popup_module(
+        title = "Welcome",
+        content = tags$b("A welcome message!", style = "color: red;")
+      ),
+      example_module()
+    )
+  )
+  app$view()
+
+  app$wait_for_idle(timeout = default_idle_timeout)
+  app$expect_no_shiny_error()
```

Comment:
What do you think about looking for the text instead?

```suggestion
  testthat::expect_equal(
    app$get_text("#landingpopup b"),
    "A welcome message!"
  )
```